specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -134,6 +134,21 @@ export function Notifications({
           count: notifications.length,
         })
       : notificationsText.notificationsLoading();
+
+  const onClearAll = () => {
+    if (notifications !== undefined) {
+      notifications.forEach((notification) => {
+        ping('/notifications/delete/', {
+          method: 'POST',
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          body: formData({ message_id: notification.messageId }),
+          errorMode: 'dismissible',
+        }).then(f.void);
+      });
+      setNotifications([]);
+    }
```

Comment:
For readability, it may be better to use a guard clause here instead of having all of the code under that if statement

```suggestion
   if (notifications === undefined) return;

    notifications.forEach((notification) => {
      ping('/notifications/delete/', {
        method: 'POST',
        // eslint-disable-next-line @typescript-eslint/naming-convention
        body: formData({ message_id: notification.messageId }),
        errorMode: 'dismissible',
      }).then(f.void);
    });
    setNotifications([]);
```